translations/TablerBundle.fr.xliff

```diff
@@ -106,6 +106,26 @@
                 <source>Take me home</source>
                 <target>Retourner au début</target>
             </trans-unit>
+            <trans-unit id="Password Reset Email Sent">
+                <source>Password Reset Email Sent</source>
+                <target>Email réinitialisation de mot de passe envoyé</target>
+            </trans-unit>
+            <trans-unit id="Enter your email address and your password will be reset and emailed to you">
+                <source>Enter your email address and your password will be reset and emailed to you</source>
+                <target>Saisissez votre adresse email et votre mot de passe sera réinitialisé par mail</target>
+            </trans-unit>
+            <trans-unit id="If an account matching your email exists, then an email was just sent that contains a link that you can use to reset your password. This link will expire in %token%.">
+                <source>If an account matching your email exists, then an email was just sent that contains a link that you can use to reset your password. This link will expire in %token%.</source>
+                <target>Si un compte correspond à l'adresse renseignée, un email vient d'être envoyé contenant un lien afin de réinitialiser votre mot de passe. Ce lien expire dans %token%.</target>
+            </trans-unit>
+            <trans-unit id="If you don't receive an email please check your spam folder or">
+                <source>If you don't receive an email please check your spam folder or</source>
```

Comment:
why does that sentence stop in the middle?

Reply:
Like we said in https://github.com/kevinpapst/TablerBundle/discussions/54 this translation is composed of a link.

So you have this translation + Try again (which is the link)